### PR TITLE
Fix summary case card issues. (PT-188001359)

### PR DESCRIPTION
**Pull request by lublagg**
Addresses the following issues:

- Bottom button font size
- If you drag over a selection of cases in a graph, then the data card view should go into summary mode, but just be a summary of those selected cases.
- If you navigate at one of the parent levels, the automatically selects all of the cases present in the child cards, so, since a selection of more than one case occurs, it should go into summary mode.

I also noticed some other bugs while working on the above that were all tied to how we were keeping track of the `summarizedCollections` and whether or not to show the summary view. Instead of having the components pass back information to the model to update the `summarizedCollections` prop on the Case Card model, `summarizedCollections` is now a view on the model that updates whenever there is a change in the selected cases.

### CODE REVIEW

File: v3/cypress/e2e/text.spec.ts
Comment (emcelroy) [inline v3/cypress/e2e/text.spec.ts:32]: Would it make sense to take out these added waits?

File: v3/cypress/support/e2e.ts
Comment (emcelroy) [inline v3/cypress/support/e2e.ts:27]: Oh, is this what got attribute button clicks working in the tests?

File: v3/src/components/case-card/card-view.scss
Comment (emcelroy) [inline v3/src/components/case-card/card-view.scss:33]: Is this the right size? The spec says 10px but then Dan said it was too small and that it "should match the font size of the rest of the text on the data card view" which is 11px. Did you and/or Dan decide it should be slightly larger? To me it looks a little too big now which is why I double checked, but maybe that's just me.

File: v3/src/components/case-card/case-attrs-view.tsx
Comment (emcelroy) [inline v3/src/components/case-card/case-attrs-view.tsx:71]: Ah :)

**Review by emcelroy — COMMENTED** [review]
This is looking good to me 👍 Changes to the card code make sense and everything seems to work well in the browser. 

I left a few comments that I don't think should hold up merging this in, but I do wonder about all the changes here. I think something got a little mixed up with the commit history. Did you maybe merge `main` into your branch? I'm not sure but think Kirk may want to clean that up before this gets merged in.

File: v3/cypress/support/e2e.ts
Comment (kswenson) [reply v3/cypress/support/e2e.ts]: I added that as part of getting cypress tests of the new formula editor to work, so I don't think it's related to the commented out tests.

**Comment by kswenson** [comment]
> Did you maybe merge `main` into your branch? I'm not sure but think Kirk may want to clean that up before this gets merged in.
> 
> There are also conflicts reported in a few files that will need to be fixed.

I merged from `main` and dealt with the conflicts.

**Review by kswenson — APPROVED** [review]
👍 Looks good -- I added a commit which merged `main` into the PR branch, simplified the `summarizedCollections()` computation, and tweaked a couple of other things. One thing to note is that if there is only one case in a collection, it needn't be summarized even if nothing is selected.

**Review by kswenson — APPROVED** [review]
👍 Changes LGTM

File: v3/cypress/e2e/text.spec.ts
Comment (kswenson) [reply v3/cypress/e2e/text.spec.ts]: I agree that the `wait`s should be removed unless they've been determined to be necessary (in which case there's probably a better solution anyway).

**Comment by kswenson** [comment]
I gave my approval, but @emcelroy's comments should still be considered/addressed.

File: v3/cypress/e2e/text.spec.ts
Comment (lublagg) [reply v3/cypress/e2e/text.spec.ts]: I didn't add these in -- these changes came in when I merged in main.

File: v3/src/components/case-card/card-view.scss
Comment (lublagg) [reply v3/src/components/case-card/card-view.scss:33]: Good catch - will change to 11px.

**Pull request merged**
